pandas/core/dtypes/common.py

```diff
@@ -120,17 +120,26 @@ def is_object_dtype(arr_or_dtype):
 
 
 def is_sparse(arr):
-    """
-    Check whether an array-like is a pandas sparse array.
+    """Check whether an array-like is a pandas sparse array.
+
+    Check that the one-dimensional array-like is a pandas sparse array.
+    Returns True if it is a pandas sparse array, not another type of
+    sparse array.
 
     Parameters
     ----------
-    arr : array-like
-        The array-like to check.
+    arr : array-like (1-D)
```

Comment:
If I'm not wrong, and based on your example, `arr` can be actually of more dimensions, but is being checked its dimensionality. Isn't it?